### PR TITLE
[JOSS] Paper nitpicks

### PR DESCRIPTION
**Pull request by RMeli**
Fixes a few references according to
> Cite a bibliography entry by referencing its identifier: [@upper1974] will create the reference “(Upper 1974)”. Omit the brackets when referring to the author as part of a sentence: “For a case study on writers block, see Upper (1974).”

Oxford postcode used `0` instead of `O` (for Oxford).

### CODE REVIEW

File: paper/paper.md [hunk 1]
```diff
@@ -26,28 +26,28 @@ affiliations:
    index: 1
  - name: Friedrich Schiller University Jena, Institute of Condensed Matter Theory and Solid-State Optics, Jena, 07743, Germany
    index: 2
- - name: Scientific Computing Department, Science and Technology Facilities Council, Rutherford Appleton Laboratory, Didcot, 0X11 0QX, UK
+ - name: Scientific Computing Department, Science and Technology Facilities Council, Rutherford Appleton Laboratory, Didcot, OX11 OQX, UK
```
Comment (RMeli) [inline paper/paper.md:29]:
```suggestion
 - name: Scientific Computing Department, Science and Technology Facilities Council, Rutherford Appleton Laboratory, Didcot, OX11 0QX, UK
```

**Comment by RMeli** [comment]
https://github.com/openjournals/joss-reviews/issues/6286

**Comment by JaGeo** [comment]
Thanks @RMeli ! 

Would it be possible for us to also thank the reviewers in the acknowledgment of the paper? Their reviews were extremely helpful.

**Review by JaGeo — APPROVED** [review]

**Comment by RMeli** [comment]
> Would it be possible for us to also thank the reviewers in the acknowledgment of the paper? Their reviews were extremely helpful.

Yes, you can certainly add the reviewers to the acknowledgment if you wish to do so!

**Pull request merged**